app/server/lib/FlexServer.ts

```diff
@@ -599,7 +599,14 @@ export class FlexServer implements GristServer {
    */
   public addBootPage() {
     if (this._check('boot')) { return; }
-    this.app.get('/boot(/(:bootKey/?)?)?$', async (req, res) => {
+    /* The regex above captures routes like
+       /boot
+       /boot/
+       /boot/KEY/
+       /boot/KEY
+       /boot//
+    */
+    this.app.get(/\/boot(\/(.*\/?)?)?$/, async (req, res) => {
```

Comment:
I think it can be even simpler, as you don't need any bootkey in your route (you just have an informative message to send):

```suggestion
    this.app.get('/boot(/*)?', async (req, res) => {
```

Reply:
Yes indeed.
Thanks :)